app/views/finance/provider/settings/show.html.slim

```diff
@@ -79,11 +79,15 @@
                   li class="string"
                     = fields.label name, label
```

Comment:
Does this result in a working label (one you can click to toggle the box?)

I'd prefer we stick to this:

<img width="237" alt="Screenshot 2021-03-23 at 12 56 18" src="https://user-images.githubusercontent.com/54224/112142761-49f5b100-8bd7-11eb-9df4-9288d2a9174f.png">
<img width="817" alt="Screenshot 2021-03-23 at 12 56 02" src="https://user-images.githubusercontent.com/54224/112142762-4a8e4780-8bd7-11eb-90a1-5e09b3638922.png">

Reply:
inline hint text suggestion:

> Make sure your Braintree account has 3D Secure enabled before you enable it here as otherwise payments initiated from the developer portal will fail.

Reply:
This is not formtastic so ... could you make a commit for that? Or add a suggestion?

Reply:
@thomasmaas this option is available for all payment gateways so the hint should be more generic or  somebody should tell me what to add in case it is not a valid hint for the payment gateway. Meaning it needs research and definition :)

Reply:
But we agreed to only show it when Braintree is selected, right? I guess you mean at the data level this option is not tight to 1 payment gateway. Working on a re-phrase.

Reply:
It is only for Braintree for now. What I meant isn't that something we want for all? Like preparing something (defining) instead of going back and forth everytime
Can you make the commit @thomasmaas ? Would be nice to